### PR TITLE
adding eslint package and config

### PR DESCRIPTION
**Pull request by laurittab**
@kalegd 

Related issue: #7

eslint

**Description**

Added ESLINT package and rules for character max, indents, semilcolons, dead code and arrow functions.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "max-len": [
+      "warn",
+      {
+        "code": 80,
+        "comments": 80,
+        "ignoreUrls": true,
+        "ignoreTemplateLiterals": true
+      }
+    ],
+    "indent": ["error", 4],
```
[the remark this answers is not in the document]
Comment (kalegd) [reply .eslintrc.json:20]: Ah I just realized I need to click `Finish your review` on GitHub for you to actually see this...

**Review by kalegd — CHANGES_REQUESTED** [review]

**Comment by laurittab** [comment]
I just saw this option, I could have just clicked that.

image

**Comment by kalegd** [comment]
> I just saw this option, I could have just clicked that.

Yea there are a ton of great features on GitHub! I'm finally trying to take the time to learn more of them myself

Anyways, thank you so much for being the first new contributor to this project 😃

**Comment by kalegd** [comment]
@laurittab would you like to be added to the README's contributor list? I've just added a section for that

**Comment by laurittab** [comment]
Yes pls that would be great 😀

**Comment by kalegd** [comment]
@all-contributors please add @laurittab for infrastructure

**Pull request merged**